cpp/src/arrow/type.cc

```diff
@@ -1257,6 +1249,17 @@ std::string MapType::ComputeFingerprint() const {
   return "";
 }
 
+std::string FixedSizeListType::ComputeFingerprint() const {
+  const auto& child_fingerprint = children_[0]->fingerprint();
+  if (!child_fingerprint.empty()) {
```

Comment:
That the child type didn't implement fingerprinting.

Reply:
Ah I see: you're bubbling the non-implemented fingerprinting.

Later, it might be worthwhile to add something like
```c++
constexpr auto kFingerprintNotImplemented = "";
```
to make this explicit
```suggestion
  if (child_fingerprint == kFingerprintNotImplemented) {
    return kFingerprintNotImplemented;
  }
```